tox.ini

```diff
@@ -43,11 +44,11 @@ setenv =
 
 
 [testenv:black]
-commands = black --skip-string-normalization --line-length=100 src/ tests/
+commands = black src/ tests/
 deps = black
 
 [testenv:lint]
-commands = flake8 src/ tests/
+commands = flake8 --ignore=W503 src/ tests/
```

Comment:
Would it be possible to ignore this for specific lines of code as in https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors or maybe fix it?